packages/components/nodes/chatmodels/AzureChatOpenAI/AzureChatOpenAI.ts

```diff
@@ -1,8 +1,9 @@
-import { AzureOpenAIInput, ChatOpenAI, OpenAIChatInput } from '@langchain/openai'
+import { AzureOpenAIInput, ChatOpenAI as LangchainChatOpenAI, OpenAIChatInput } from '@langchain/openai'
```

Comment:
I'm getting access to Azure Vision model, in the meantime we can get rid of this first, and create another PR once we have access